lib/jose-utils.js

```diff
@@ -367,5 +367,15 @@ Utils.sha256 = function(str) {
 Utils.isCryptoKey = function(rsa_key) {
   // Some browsers don't expose the CryptoKey as an object, so we need to check
   // the constructor's name.
-  return rsa_key.constructor.name == 'CryptoKey';
+  if (rsa_key.constructor.name == 'CryptoKey') {
```

Comment:
Then I would have to say "no". CryptoKey is going to be defined in WebKit, which would mean that I can't do what I need to do in terms of feeding js-jose homebrew CryptoKeys that support algorithms not supported by native browser crypto.